pkgs/by-name/da/das/package.nix

```diff
@@ -5,24 +5,24 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "das";
-  version = "0.3.8";
-  format = "pyproject";
+  version = "1.0.3";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "snovvcrash";
     repo = "DivideAndScan";
     rev = "refs/tags/v${version}";
-    hash = "sha256-a9gnEBTvZshw42M/GrpCgjZh6FOzL45aZqGRyeHO0ec=";
+    hash = "sha256-3d2sYxht4AMxz9K7Hm9VYWJ0bTfVFeGZFC+wGwteXCQ=";
```

Comment:
I'm getting a mismatch:

``bash
error: hash mismatch in fixed-output derivation '/nix/store/0w7lld3vrla8l63jkgaqjlgajpmah37r-source.drv':
         specified: sha256-3d2sYxht4AMxz9K7Hm9VYWJ0bTfVFeGZFC+wGwteXCQ=
            got:    sha256-WZmWpcBqxsNH96nVWwoepFhsvdxZpYKmAjNd7ghIJMA=
```